introduction/architecture.md

```diff
@@ -110,53 +96,52 @@ while maintaining support for multiple indexes.
 For more on the motivation and design of TimescaleDB, please see our
 [technical blog post][chunking].
 
-## TimescaleDB Clustering (PRIVATE BETA) [](timescaledb-clustering)
+## Distributed Hypertables (PRIVATE BETA) [](distributed-hypertables)
+
+>:WARNING: Distributed hypertables are currently in PRIVATE BETA.
+This feature is not meant for production use. For more information,
+please [contact us][contact].
 
->:WARNING: Running TimescaleDB in a multi-node setup is currently in PRIVATE BETA.
-This method of deployment is not meant for production use. For more information,
-please
-[contact us][contact].
+TimescaleDB supports distributing hypertables across multiple nodes
+(i.e., a cluster) by leveraging the same hypertable and chunk
+primitives as described above. This allows TimescaleDB to scale
+inserts and queries beyond the capabilities of a single TimescaleDB
+instance.
 
-TimescaleDB supports multi-node clustering by leveraging the hypertable and chunk primitives described above.
-In the multi-node topology, all nodes are TimescaleDB instances, but may play different roles
-depending on their responsibilities within a distributed database.
+Distributed hypertables look very similar to regular hypertables, with
+the main difference being that chunks are not stored locally. There
+are also some features of regular hypertables that distributed
+hypertables do not support (see section on current limitations).
 
-### Node Types
+### Distributed Databases and Nodes
+
+A distributed hypertable exist in a *distributed database* that
+consists of multiple databases stored across one or more TimescaleDB
+instances. A database that is part of a distributed database can
+assume one of two *node* roles (but not both):
 
-The two roles that nodes can take on are:
 1) Access node
 2) Data node
 
-A access node accepts all requests from the client, distributes the requests and queries
+A client connects to an access node database. It then distributes the requests and queries
```

Comment:
The "It" here can refer to either the subject or the object. It looks like it is referring to the client, not the access node.